todo/test/util/defineclass.spec.js

```diff
@@ -0,0 +1,118 @@
+import {defineClass} from '../../src/util';
+
+describe('defineClass', () => {
+  it('defineClass 함수이다.', () => {
+    expect(defineClass instanceof Function).toBe(true);
+  });
+
+  it('메서드는 클래스를 생성한다.', () => {
+    const Animal = defineClass();
+
+    expect(Animal.prototype instanceof Object).toBe(true);
+
+    const animal = new Animal();
+
+    expect(animal instanceof Animal).toBe(true);
+  });
+
+  describe('첫 번째 인수로는 자식 클래스 객체를 전달한다.', () => {
+    it('자식 클래스 객체에 init 메서드가 존재하면 인스턴스를 생성할 때 호출된다.', () => {
+      const initializer = jasmine.createSpy('initializer');
+      const Animal = defineClass({
+        init: initializer
+      });
+      const animal = new Animal();
+
+      expect(animal instanceof Animal).toBe(true);
+      expect(initializer).toHaveBeenCalledTimes(1);
+    });
+
+    it('자식 클래스 객체에 init 메서드 안에서는 생성된 인스턴스를 this를 통해 접근 가능하다.', () => {
+      const Animal = defineClass({
+        init() {
+          this.value = 1;
+        }
+      });
+      const animal = new Animal();
+
+      expect(animal.value).toEqual(1);
+    });
+
+    it('자식 클래스 객체에 프로퍼티를 추가로 전달하면 프로토타입 메서드로 추가된다.', () => {
+      const initializer = jasmine.createSpy('initializer');
+      const move = jasmine.createSpy('move');
+
+      const Animal = defineClass({
+        init: initializer,
+        move
+      });
+      const animal = new Animal();
+      animal.move();
+
+      expect(initializer).toHaveBeenCalledTimes(1);
+      expect(move).toHaveBeenCalledTimes(1);
+    });
+  });
+
+  describe('두번째 인수로는 부모 클래스를 전달한다.', () => {
```

Comment:
```suggestion
  describe('두번째 인수로는 부모 클래스의 생성자 함수를 전달한다.', () => {
```
약간 더 명확하게 해주는게 어떨까 해서요 ㅎㅎ

Reply:
넵 수정했습니다!